tests/PerformanceTests/Microsoft.Identity.Web.Perf.Benchmark/TokenAcquisitionTests.cs

```
@@ -0,0 +1,108 @@
+    public class TokenAcquisitionTests
+    {
+        private WebApplicationFactory<Startup> _factory;
+        private HttpClient _client;
+
+        public TokenAcquisitionTests()
+        {
+            _factory = new WebApplicationFactory<Startup>();
+        }
+
+        [GlobalSetup]
+        public void GlobalSetup()
+        {
+            CopyDependencies();
+            _client = _factory
+            .CreateClient(new WebApplicationFactoryClientOptions
+            {
+                AllowAutoRedirect = false,
+            });
+            string accessToken = AcquireTokenForLabUserAsync().GetAwaiter().GetResult().AccessToken;
+            _client.DefaultRequestHeaders.Add(
+                "Authorization",
+                string.Format(
+                    CultureInfo.InvariantCulture,
+                    "{0} {1}",
+                    "Bearer",
+                    accessToken));
+        }
+
+        [GlobalCleanup]
+        public void GlobalCleanup()
+        {
+            _client.Dispose();
+            _factory.Dispose();
+        }
+
+        [Benchmark]
+        public void GetAccessTokenForUserAsync()
+        {
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, TestConstants.SecurePageGetTokenForUserAsync);
+            HttpResponseMessage response = _client.SendAsync(httpRequestMessage).GetAwaiter().GetResult();
+            if (!response.IsSuccessStatusCode)
+            {
+                throw new Exception("Failed.");
+            }
+        }
+
+        [Benchmark]
+        public void GetAccessTokenForAppAsync()
+        {
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage(HttpMethod.Get, TestConstants.SecurePageGetTokenForAppAsync);
+            HttpResponseMessage response = _client.SendAsync(httpRequestMessage).GetAwaiter().GetResult();
+            if (!response.IsSuccessStatusCode)
+            {
+                throw new Exception("Failed.");
+            }
```

do you want to return the error code? maybe include which test this is for?